### PR TITLE
kind:: improve kind deployment scripts

### PR DESCRIPTION
**Pull request by hedibouattour**

### CODE REVIEW

File: test/kind/Makefile [hunk 1]
```diff
@@ -16,3 +16,37 @@ dev:
 	docker push localhost:5000/calicovpp/agent:$(TAG)
 	docker image tag calicovpp/multinet-monitor:$(TAG) localhost:5000/calicovpp/multinet-monitor:$(TAG)
 	docker push localhost:5000/calicovpp/multinet-monitor:$(TAG)
+
+
+.PHONY: load
+load:
+	docker pull docker.io/calico/node:master
+	docker save docker.io/calico/node:master -o /home/hbouatto/images/node
```
Comment (sknat) [inline test/kind/Makefile:24]: We could probably either `mkdir $SCRIPTDIR/.imagecache/` and use this as a cache directory

File: test/kind/Makefile [hunk 2]
```diff
@@ -1,8 +1,8 @@
 TAG ?= latest # Tag images with :$(TAG)
 
-.PHONY: new-cluster
+.PHONY: kind-new-cluster
```
Comment (sknat) [inline test/kind/Makefile:3]: in this makefile, it's still `make new-cluster` right ?

**Review by sknat — COMMENTED** [review]
that's really useful, thanks 😄

File: test/kind/Makefile (hunk 1 above)
Comment (hedibouattour) [reply test/kind/Makefile:24]: sorry for that personal home 😂

Comment (sknat) [reply test/kind/Makefile:24]: 😄 np, did the same more than a couple times

**Review by sknat — APPROVED** [review]

**Pull request merged**